### PR TITLE
Lightcleanups

### PR DESCRIPTION
**Pull request by OznOg**

### CODE REVIEW

File: parameter/Element.h [hunk 1]
```diff
@@ -76,12 +76,31 @@ class CElement : public IXmlSink, public IXmlSource
...
+     *
+     * Note: this method will assert if given a wrong child index (>= number of children)
+     *
+     * @param[in] uiIndex the index of the child element from 0 to number of children - 1
+     * @return the child element
+     */
+    const CElement* getChild(size_t uiIndex) const;
+
+    /**
+     * Get a child element (const)
```
Comment (dawagner) [inline parameter/Element.h]: this method isn't const (bad copy-paste ? ;))

File: parameter/XmlFileIncluderElement.cpp [hunk 2]
```diff
@@ -120,9 +119,9 @@ std::string CXmlFileIncluderElement::getIncludedElementType() const
 {
     std::string strKind = getKind();
 
-    int iPosToRemoveFrom = strKind.rfind("Include", -1);
+    std::string::size_type uiPosToRemoveFrom = strKind.rfind("Include", std::string::npos);
```
Comment (dawagner) [inline parameter/XmlFileIncluderElement.cpp]: There are some inconsistencies: in most places, the "ui" prefix has been kept but where there previously was a "i" prefix (i.e. in RuleParser), the "i" prefix was removed.

**Comment by krocard** [comment]
:-1: Could you use a meaningful name for your pull request ? You already used "Lightcleanups" last time. Could you also add a description ?

**Comment by dawagner** [comment]
Please rebase onto master: some of these commits are already merged.

**Comment by dawagner** [comment]
obsoleted by #83

**Pull request closed without merging**